### ISSUE TITLE
filter/ignore user by id or pattern

### ISSUE THREAD

**Issue by NikkyAI** [Emberwalker/Thumpcord#6]
in our usecase we have Thump and ThumpCord set up
and a irc-discord link as well (named 'cord')
when somone speaks in irc it will be both picked up by Thump and cord and then by ThumpCord
so ignoring users or ignoring messages matching a regex would be very useful
eg: /^<.+>.+/

**Comment by lopcode** [comment]
@Emberwalker Filtering by name exists in the IRC plugin already, in MessageHandler.kt - could bring that to the Discord plugin first and then maybe try regex?

**Comment by Wyldern** [comment]
Yeah it should be an easy change. May not be able to get to it quickly though - lot of uni work to handle right now! So yes to the feature, but it may be a while to do it.
